scripts/core/editor3/reducers/toolbar.jsx

```diff
@@ -153,8 +153,8 @@ export const addMedia = (editorState, media) => {
  * @param {Object} data Contains the entityKey and the new image data.
  * @description Updates the given entityKey with the new image data.
  */
-const updateImage = (state, {entityKey, img}) => {
-    Entity.replaceData(entityKey, {img});
+const updateImage = (state, {entityKey, media}) => {
+    Entity.replaceData(entityKey, {media});
```

Comment:
Please do not use the `Entity` API. It is being deprecated in 0.11 and it will be hard to update DraftJS afterwards. Use `contentState.replaceEntityData` instead. Check the usage in `processCells` or other functions in this file.